### ISSUE TITLE
Time axis on "Last 30 days report" doesn't make sense

### ISSUE THREAD

**Issue by fgrueninger** [mysteriumnetwork/node#5234]
**Describe the bug**
The time axis on the 30 days report doesn't make sense; it doesn't contain valid dates.

**To Reproduce**
1. Open your dashboard

**Expected behavior**
A time axis that makes sense.

**Screenshots**
![image](https://user-images.githubusercontent.com/33499883/174896122-346f7c9c-ff81-4536-a33f-0242ee3a48cc.png)

**Environment (please complete the following information):**
- Node version: 1.8.1-b39ec047
- NodeUI version: 1.16.2

**Comment by n10ty** [comment]
Have the same
```
Node version: mainnet (1.9.2-a5cea4f0)
NodeUI version: 1.16.2
```

**Comment by pvdl** [comment]
Not yet fixed in:
```Node version: mainnet (1.9.5-8569a6fd)
NodeUI version: 1.17.1```

**Comment by n10ty** [comment]
#5357

**Comment by pvdl** [comment]
Fixed in `1.12.1-e40f2b25`
